openapi-format.js

```diff
@@ -1135,9 +1135,8 @@ async function openapiConvertVersion(oaObj, options) {
 
         // Change type > single enum
         node = convertConst(node);
-
-        this.update(node);
       }
+      this.update(node);
```

Comment:
@thim81 i'm not sure if moving this outside the `if (node.type)` is a good idea (but necessary here since it needs to operate on nodes that don't have `type`). 
If it helps performance we could add `anyOf` and `oneOf` to the `if (node.type)` condition instead? Felt a bit clunky to me tho: `if (node.type || node.anyOf || node.oneOf)`. Happy to go with whatever works for you though.

Reply:
I think it is fine. The performance does not seem to degrade.